lib/utils/server/feature-flags.ts

```diff
@@ -1,6 +1,6 @@
 import * as Sentry from "@sentry/nextjs";
 
-export type FeatureFlag = "contributions_evolution_by_type" | "star_search" | "starsearch-workspaces" | "oscr-rating";
+export type FeatureFlag = "contributions_evolution_by_type" | "oscr-rating";
```

Comment:
I also removed the `star_Search` feature flag that should have been removed a while ago.